### DISCUSSION TITLE
Upgrading Postgres versions

### DISCUSSION THREAD

**Discussion by detj** [supabase/supabase#14783] [General]
As I understand, Supabase is using Postgres v13 as of today. I have a few related questions.

1. What is Supabase's Postgres upgrade policy? When and how frequently will Supabase make the decision to upgrade Postgres version?
2. Will there be downtime when such upgrades happen? Has Supabase performed such upgrades in the past? As a customer of Supabase, do I have to take any actions/precautions before hand?

Please share links to documentation or related blog post articles if you can. Would be super helpful. Thanks.

**Comment by GaryAustin1** [comment]
New instances use Postgres 15.  If on free plan if you pause restore you will get Postgres 15.  On pro plans you have to request upgrade from support.

I can't comment on the impact of moving from 13 to 15, if there is any backward compatibility issues, but in general Postgres upgrades seem pretty painless.

**Comment by cmantsch** [comment]
How does this work for selfhosted instances? We are currently on Postgres 14 but would like to upgrade to 15, is there a best practice how to do that for supabase?
